source/documentation/basic-project.md

```diff
@@ -81,6 +81,62 @@ generating the site.
 With this option passed, `{{ site.url }}/about` will now be generated as
 `http://my.dev.host/blog-skelton/output_dev/about` instead of `/about`.
 
+## CSS and JavaScript
+
+The [Blog Skeleton](https://github.com/sculpin/sculpin-blog-skeleton) project
+uses [Symfony Webpack Encore](https://symfony.com/doc/current/frontend.html) to
+manage assets. If you're setting up your project with Sculpin directly, not
+using the skeleton, install Encore using the
+[setup instructions](https://symfony.com/doc/current/frontend/encore/installation.html).
+
+The rest of this section assumes that you are using Encore.
```

Comment:
i think we need to link to the encore config file of the skeleton as well, otherwise the things below are not correct. maybe add here a second part...  "... assumes that you are using Encore, with a configuration like [in the skeleton](link to skeleton encore config file)".

Reply:
Thanks, added that.